Codeinwp/optimole-wp
# feat: add logging for offloading/rollback

### All Submissions:

 

### Changes proposed in this Pull Request:
This PR adds offloading/rollback logs to Optimole's Offloading/Rollback process. Logs are saved into uploads but do not contain any private information. We also do not use image URLs in the logs, just the ID to avoid exposing possibly private URLs.

Logs are also sent to Axiom.

Part of https://github.com/Codeinwp/optimole-service/pull/1104
 
<img width="724" alt="Screenshot 2023-10-05 at 2 19 51 PM" src="https://github.com/Codeinwp/optimole-wp/assets/2649903/b65e4cfb-f2a8-42d7-aeb4-456385b976c7">

Closes https://github.com/Codeinwp/optimole-service/issues/1083.

### How to test the changes in this Pull Request:

1. Do a rollback or offload.
2. Confirm the logs are displaying properly.
3. If setup, confirm logs appear fine in Axiom.

### Other information:
